### ISSUE TITLE
Color picker css rule should be removed from the stylesheet if the option is reset to default

### ISSUE THREAD

**Issue by chetzof** [gambitph/Titan-Framework#233]
```
 [
        'name'=> 'Header background color',
        'id' => 'header_bg',
        'type' => 'color',
        'css' => '.navbar-main { background-image: $header_bg; }',
    ]
```

After color picker option is set to some value and then reset to default value (false) , in stylesheet an old rule remains, it does not get removed.

**Comment by ori** [comment]
Same. Guys, fix it as soon as possible. We've chosen your framework for our premium theme, and there're lots of bugs!

**Comment by bfintal** [comment]
Verified fixed in 1.9
